src/pe_reports/report_gen/templates/report_gen_UI/home_report_gen.html

```diff
@@ -0,0 +1,190 @@
+          Generate Reports
+        </div>
+        <div class="card-body">
+          <form method="post">
+            {{ form_external.hidden_tag() }} {{
+            form_external.report_date.label(style='font-size: 24px') }} {{
+            form_external.report_date(class='form-control') }} {{
+            form_external.output_directory.label(style='font-size: 24px') }} {{
+            form_external.output_directory(class='form-control' ,
+            value="/var/www/current_report_run" ) }}
+            <br />
+            {{ form_external.submit(class='btn btn-primary') }}
+          </form>
+        </div>
+      </div>
+    </div>
+  </div>
+  <div
+    id="BulletinDiv"
+    class="row justify-content-lg-center"
+    style="display: none"
+  >
+    <div class="d-flex justify-content-center">
+      <div class="card" id="content">
+        <div class="card-header text-white bg-primary mb-3">
+          Generate Cybersix Bulletin
+        </div>
+        <div class="card-body">
+          <form method="post">
+            {{ bulletin_form.hidden_tag() }} {{
+            bulletin_form.cybersix_id.label(style='font-size: 24px') }} {{
+            bulletin_form.cybersix_id(class='form-control') }} {{
+            bulletin_form.user_input.label(style='font-size: 24px') }} {{
+            bulletin_form.user_input(class='form-control') }} {{
+            bulletin_form.output_directory1.label(style='font-size: 24px') }} {{
+            bulletin_form.output_directory1(class='form-control' ,
+            value="/var/www/cybersix_bulletins" ) }}{{
+            bulletin_form.file_name.label(style='font-size: 24px') }} {{
+            bulletin_form.file_name(class='form-control') }}
+            <br />
+            {{ bulletin_form.submit1(class='btn btn-primary') }}
+          </form>
+        </div>
+      </div>
+    </div>
+  </div>
+  <div
+    id="CredsDiv"
+    class="row justify-content-lg-center"
+    style="display: none"
+  >
+    <div class="d-flex justify-content-center">
+      <div class="card" id="content">
+        <div class="card-header text-white bg-primary mb-3">
+          Generate Credential Breach Bulletin
+        </div>
+        <div class="card-body">
+          <form method="post">
+            {{ creds_form.hidden_tag() }} {{
+            creds_form.org_id.label(style='font-size: 24px') }} {{
+            creds_form.org_id(class='form-control') }} {{
+            creds_form.breach_name.label(style='font-size: 24px') }} {{
+            creds_form.breach_name(class='form-control' ) }}
+            <br />
+            {{ creds_form.submit2(class='btn btn-primary') }}
+          </form>
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
+
+<div id="loading" class="justify-content-lg-center"></div>
+<div class="row justify-content-lg-center">
+  <div class="row col-lg-6 offset-lg-3">
+    <input type="hidden" value="{% block title %} Stakeholder {% endblock%}" />
```

Comment:
Use consistent spacing:
```suggestion
    <input type="hidden" value="{% block title %} Stakeholder {% endblock %}" />
```